servicecontrol/transport-adapter.md

```diff
@@ -0,0 +1,110 @@
+
+In this case a separate transport adapter is deployed for each instance of a broker (e.g. RabbitMQ instance, SQL Server instance or ASB/ASQ namespace) while ServiceControl connects to its own instance. The adapters forward messages between instances.
+
+Following code shows the configuration of the Transport Adapter in the multi-instance scenario:
+
+snippet: MultiInstance
+
+Notice that both adapter configurations use the same connection string for the ServiceControl and a different connection string for the endpoint-side transport.
+
+NOTE: Some transports, such as [SQL Server transport](/nservicebus/sqlserver/), offer a native multi-instance mode. This mode is considered an advanced routing feature and allows use of a single instance of the transport adapter even though there are multiple instances of the broker.
+
+
+## Queue configuration
+
+The transport adapter allows configuration of the addresses of the forwarded queues: audit, error and control (input queue of ServiceControl). Both endpoint- and ServiceControl-side queues can be configured. 
+
+NOTE: The default values are only suitable in cases where both sides of the adapter use different transports or at least different broker instances. In case the adapter runs on a single transport and instance, the queue names on one side need to be altered.
+
+snippet: AuditQueues
+
+snippet: ErrorQueues
+
+snippet: ControlQueues
+
+
+### Poison
+
+The poison message queue is a special queue which is used when messages cannot be received from the transport because they are corrupted (e.g. message headers are malformed).
+
+snippet: PoisonQueue
+
+Such messages cannot be forwarded to the error queue because ServiceControl won't be able to receive them. During normal operations the NServiceBus system never generates such malformed messages. They can be a generated, for example, by a misbehaving integration component.
+
+
+## Failure handling
+
+ServiceControl Transport Adapter uses different failure handling strategies for different types of messages.
+
+
+### Audits and errors
+
+Audited and failed messages sent by the endpoints to `audit` and `error` queues are subject to a *retry forever* policy. This means that the adapter will retry forwarding such message to ServiceControl until it succeeds. 
+
+
+### Control messages
+
+Control messages, such as heartbeats and custom checks, are subject to a *best effort* policy which retries each message up to a configured number of times.
+
+snippet: ControlRetries
+
+If the adapter still cannot forward the message, it will be dropped.
+
+
+### Retry messages
+
+Retry messages are messages selected for retry processing in their destination endpoint via ServicePulse. Retry messages use an immediate retry mechanism similar to control messages but in case of failure they are routed back to ServiceControl and reappear in ServicePulse. The following API controls the maximum number of attempts to forward a retry message:
```

Comment:
>  Retry messages use an immediate retry mechanism similar to control messages but in case of failure they are routed back to ServiceControl and reappear in ServicePulse.

ServiceControl will attempt to retry failed messages for the configured number of times. If messages processing still fails, the messages will be routed back to ServiceControl and will reappear in ServicePulse.